packages/core/src/js/integrations/reactnativeerrorhandlers.ts

```diff
@@ -81,6 +81,7 @@ function attachUnhandledRejectionHandler(): void {
         data: { id },
         originalException: error,
         syntheticException: isErrorLike(error) ? undefined : createSyntheticError(),
+        mechanism: { handled: false, type: 'onunhandledrejection' },
```

Comment:
We have to mark the rejection `handled: true` to avoid breaking crash rate calculations.

Rejections do not crash the app in RN at the moment.

https://github.com/getsentry/sentry-react-native/issues/4141

Reply:
Changed from false to true on https://github.com/getsentry/sentry-react-native/pull/4457/commits/36f0f3af6c619f88d45838aa47d36ad24f74f9d9

This flag always confused me on other SDKs since depending on the platform an unhandled error could crash or not an app.

Reply:
Yeah, it usage is less than ideal.